src/_pytest/outcomes.py

```diff
@@ -113,6 +113,7 @@ def exit(
 
     :param returncode:
         Return code to be used when exiting pytest.
+    :default returncode: None
```

Comment:
IIUC, the problem was not that it was unclear that `returncode` might be `None`, the source of confusion was "what None *means*", so I think this should instead be something like:

```suggestion
    :param returncode:
        Return code to be used when exiting pytest. None means the same as ``0`` (no error), same as :func:`sys.exit`.
```